assets/css/style.css

```diff
@@ -36,8 +36,8 @@ h1, h2, h3, h4, h5, h6 {
   }
 }
 
-/*--------------------------------------------------------------
-# Preloader
+/*--------------------------------------------------------------
```

Comment:
do you know how / why these parts changed?

Reply:
Yeah, I think it was the Prettier code formatter. I actually noticed it was different than what you were doing so I tried not to format it but I think I have it set to format on save and I forgot.

Reply:
I think I jumped to the wrong conclusion. When I went to try and fix it, I saw this error. "We’ve detected the file has mixed line endings. When you commit changes we will normalize them to Windows-style" While trying to edit your original CSS file on Github. So I guess our editors differ in how they handle line endings. I don't know a lot about that but, GitHub doesn't show me the error when trying to edit my file so, I think it might be your editor that is creating non-standard files.